src/metadata/metadata_io.c

```diff
@@ -54,6 +54,7 @@ enum ocf_mio_size {
 	ocf_mio_size_16,
 	ocf_mio_size_32,
 	ocf_mio_size_64,
+	ocf_mio_size_128,
```

Comment:
This unnecessarily increases size of mio mpool. I'd rather suggest to make following change in the limits definition:
```c
uint32_t limits[] = {
        [0 ... MIO_RPOOL_THRESHOLD - 1] = -1,
        [MIO_RPOOL_THRESHOLD ... ocf_mio_size_max - 1] = MIO_RPOOL_LIMIT,
        [ocf_mio_size_max ... env_mpool_max] = -1,
};
```

Reply:
resolved :)